sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -97,6 +98,10 @@ def _format_url_section(template, **kwargs):
             formatted_components = template.split("/")
             components = [c for c in formatted_components if "{{{}}}".format(key.args[0]) not in c]
             template = "/".join(components)
+            if last_template == template:
+                return template
```

Comment:
Have we tested this with Storage?

Reply:
We have storage tests in our pipeline. So I guess the answer is yes?